R/derive_interp_records.R

```diff
@@ -40,16 +47,13 @@
 #'     AGE = round(AGE * 30.4375)
 #'   ) %>%
 #'   # Interpolate the AGE by SEX
-#'   group_by(SEX) %>%
 #'   # Ensure first that Age unit is "DAYS"
```

Comment:
Comment `# Ensure first that Age unit is "DAYS"` belongs above line 47

Reply:
Comment moved up